schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/history/HistoryHelper.java

```diff
@@ -52,6 +56,30 @@ public History execute(Transaction tx) throws ModelException {
         });
     }
 
+    private static User getUser(Key userId) {
+        try {
+            if (userId != null) {
+                UserDao userDao = new UserDao();
+                return userDao.get(userId);
```

Comment:
Are you sure, getOrNull calls AsyncDatastoreDelegate.getAsync(Transaction tx, Key id) which has a check to see if key == null and throws NullPointerException if it is